server/routes/private.js

```diff
@@ -48,10 +48,21 @@ router.post('/book-appointment', async (req, res) => {
 
     if (!_.isNil(error)) res.send(error);
 
-    // Attempts to insert the appointment into the database.
-    const isSuccessfullyInserted = await appointmentHandler.bookAppointment(studentId, workerTimeslotId, purpose, studentNotes, workerComments);
+    // Checks worker timeslot status, either available or unavailable)
 
-    res.send(isSuccessfullyInserted);
+    const workerIsAvailable = await workerTimeslotHandler.checkWorkerAvailability(workerTimeslotId);
+
+    If(workerIsAvailable == true); {
```

Comment:
Hi Nancy,
To prevent runtime errors, I would recommend adjusting this line to be `if (workerIsAvailable) {` to remove the semicolon and unneeded verification (as `checkWorkerAvailability` already returns a boolean.

Reply:
Hi Amy,
Thank you for your suggestion!
My Action: I have made the recommended changes to line 55.

You can see this change in line 55 of server/routes/private.js

Please confirm if this takes care of your suggestion!